### PR TITLE
Application error handling

### LINKED ISSUE #424: As a developer I want to be notified of Application specific errors
### Description

As a developer, I want to distinguish between code specific and application specific errors.

Image

### Acceptance Criteria

- ability to run a test to generate an application specific error

### PR DESCRIPTION
**Pull request by MrDSGC**
Closes: #424

## Description

- implemented application error toast
- removed notification toast (not used anymore)
- added global constants file

image

image

______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md).
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: src/providers/Project/index.tsx [hunk 1]
```diff
@@ -192,7 +195,8 @@ export const ProjectProvider: React.FC<ProjectProviderProps> = ({
     } catch (e) {
       console.error(e);
       setIsSaving(false);
-      showError(e?.message);
+      setApplicationErrorMessage(translateError(e?.message));
+      setShowErrorToast(true);
```
Comment (bthaile) [inline src/providers/Project/index.tsx]: Are there Application Error message that should not show in a toast?
If not should show error when `applicationErrorMessage` has a value. Prob don't need `showErrorToast`

File: src/components/CadenceEditor/Notifications/index.tsx [hunk 2]
```diff
@@ -37,7 +37,9 @@ const Notifications = () => {
 
   // EFFECTS  -------------------------------------------------------------------
   useEffect(() => {
+    console.log('acct', project.accounts);
     setProjectAccounts((prevAccounts) => {
+      console.log('prev', prevAccounts);
```
Comment (bthaile) [inline src/components/CadenceEditor/Notifications/index.tsx]: Might not need these console logs.

File: src/components/Editor/ErrorToastContainer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+  return (
+    <Flex sx={styles.root}>
+      {showErrorToast && (
+        <ErrorToast>
+          <Flex sx={styles.errorContent}>
+            <Box sx={styles.message}>
+              {applicationErrorMessage.length > 0
+                ? applicationErrorMessage
+                : `this is a really long filler text to show overflow properties of the application toast that wont be a part of the regular error messageOops! Something went wrong. If you'd like, please report the Bug. Thank you!`}
```
Comment (bthaile) [inline src/components/Editor/ErrorToastContainer/index.tsx]: is this really long message needed ?

File: src/providers/Project/index.tsx (hunk 1 above)
Comment (MrDSGC) [reply src/providers/Project/index.tsx]: https://dapperlabs.slack.com/archives/C03UWGN8M0Q/p1670610305007239?thread_ts=1670609513.928649&cid=C03UWGN8M0Q error messaging will depend on what we discussed here

Comment (bthaile) [reply src/providers/Project/index.tsx]: not what I'm asking.
displaying an error toast is useing two properties. `applicationErrorMessage` and `showErrorToast`
Show not show a toast when ever applicationErrorMessage is set. Then you don't need `showErrorToast` property. So my question is, Are application messages always shown as toasts?

**Review by bthaile — APPROVED** [review]
lgtm

**Review by bthaile — APPROVED** [review]
changes look good.

**Pull request merged**